### PR TITLE
pkp/pkp-lib#1347 display users import xml validation errors

### PR DESCRIPTION
**Pull request by bozana**
S. e) here: https://github.com/pkp/pkp-lib/issues/1347
also for the native import

### CODE REVIEW

File: plugins/importexport/native/templates/results.tpl [hunk 1]
```diff
@@ -8,15 +8,22 @@
...
-	{foreach from=$content item=contentItem}
-		<li>
-			{if is_a($contentItem, 'Submission')}
-				{$contentItem->getLocalizedTitle()|strip_unsafe_html}</li>
-			{else}
-				{$contentItem->getIssueIdentification()|escape}
-			{/if}
-		</li>
+{if $validationErrors}
+	<h2>Validation errors:</h2>
```
Comment (asmecher) [inline plugins/importexport/native/templates/results.tpl]: Hard-coded English text.

Comment (bozana) [inline plugins/importexport/native/templates/results.tpl]: may i put the key in the manager.xml in the pkp-lib? -- it will be then used by omp too...

Comment (asmecher) [inline plugins/importexport/native/templates/results.tpl]: Yes, I think that's a good place for it.

**Comment by bozana** [comment]
@asmecher, could you please review these PRs? The OMP equivalent is coming too :-) OK?

File: plugins/importexport/native/templates/results.tpl [hunk 2]
```diff
@@ -8,15 +8,22 @@
...
-			{if is_a($contentItem, 'Submission')}
-				{$contentItem->getLocalizedTitle()|strip_unsafe_html}</li>
-			{else}
-				{$contentItem->getIssueIdentification()|escape}
-			{/if}
-		</li>
+{if $validationErrors}
+	<h2>{translate key="plugins.importexport.common.validationErrors"}</h2>
+	{foreach from=$validationErrors item=validationError}
+		<p>{$validationError->message|escape}</p>
```
Comment (asmecher) [inline plugins/importexport/native/templates/results.tpl]: This would probably be better presented as `<li>` elements within a `<ul>`.

**Comment by asmecher** [comment]
When the PRs are ready, summarize them in a comment on the issue and then assign the issue over to me.

**Comment by bozana** [comment]
Shall I try to fix this JS test error?

**Comment by asmecher** [comment]
@NateWr is working on it, I think.

**Pull request merged**